Dockerfile

```diff
@@ -1,36 +1,30 @@
-## Image to build from sources
-
-FROM debian:buster
+FROM python:3.7-slim-buster AS compile-image
 LABEL maintainer="Wazo Maintainers <dev@wazo.community>"
 
-ENV DEBIAN_FRONTEND noninteractive
-ENV HOME /root
-
-# Add dependencies
-RUN apt-get -qq update
-RUN apt-get -qq -y install \
-    git \
-    apt-utils \
-    python3-pip \
-    python3-dev
-
-# Install wazo-dxtora
-ADD . /usr/src/dxtora
-WORKDIR /usr/src/dxtora
-RUN pip3 install -r requirements.txt
-RUN python3 setup.py install
-
-# Configure environment
-RUN mkdir /var/lib/wazo-dxtora \
-    && touch /var/log/wazo-dxtora.log \
-    && adduser --quiet --system --group --no-create-home wazo-dxtora \
-    && chown wazo-dxtora:wazo-dxtora /var/log/wazo-dxtora.log \
-    && install -d -o wazo-dxtora -g wazo-dxtora /run/wazo-dxtora \
-    && cp -r etc/wazo-dxtora /etc
-
-# Clean
-WORKDIR /root
-RUN rm -rf /usr/src/dxtora
-RUN apt-get clean
+RUN python -m venv /opt/venv
+# Activate virtual env
+ENV PATH="/opt/venv/bin:$PATH"
+
+WORKDIR /usr/src/wazo-dxtora
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY setup.py setup.py
+COPY wazo_dxtora wazo_dxtora
+RUN python setup.py install
+
+FROM python:3.7-slim-buster AS build-image
+COPY --from=compile-image /opt/venv /opt/venv
+
+COPY ./etc/wazo-dxtora /etc/wazo-dxtora
+
+RUN true \
+    && adduser --quiet --system --group --home /var/lib/wazo-dxtora wazo-dxtora \
+    && install -o wazo-dxtora -g wazo-dxtora /dev/null /var/log/wazo-dxtora.log \
+    && install -d -o wazo-dxtora -g wazo-dxtora /run/wazo-dxtora
+
+EXPOSE 9500
```

Comment:
I'm not sure I understand why we need to expose this port

Reply:
We don't, it's gone :) Thanks!